about_our_class.md

```diff
@@ -1,9 +1,5 @@
-# About Our Class
-As a class exercise, we will each use git and GitHub to add our own little autobiography to this file!
```

Comment:
Please add the header lines back to the file, it'll work as a sort of title header for the whole thing.